config/samples/config_v1alpha1_certstore_akv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: config.ratify.deislabs.io/v1alpha1
+kind: CertificateStore
+metadata:
+  name: certstore-akv
+spec:
+  provider: azurekeyvault
+  parameters:
+    keyvaultName: sample
+    certificates:  |
+      array:
+        - |
+          certificateName: yourCertName
```

Comment:
Hi @fseldow , please review when convenient, I still need to pipe through the chart changes. but these are some sample CRDs